deploy/manifests/alpha/basic.yml

```diff
@@ -0,0 +1,15 @@
+---
+applications:
+- name: alpha-basic
+  memory: 1024M
+  instances: 2
+  buildpack: java_buildpack
+  health-check-http-endpoint: /records
+  random-route: true
+  services:
+    - alpha-db
```

Comment:
We currently have our DB connection strings defined in `config.yaml` - we will be keeping this I imagine and updating it to the new connection strings provided by PaaS? I imagine that here we're only saying that this particular application has access to this particular database?